sound/soc/sof/ipc4-topology.c

```diff
@@ -2107,12 +2107,17 @@ static int sof_ipc4_control_load_volume(struct snd_sof_dev *sdev, struct snd_sof
 	msg->primary |= SOF_IPC4_MSG_DIR(SOF_IPC4_MSG_REQUEST);
 	msg->primary |= SOF_IPC4_MSG_TARGET(SOF_IPC4_MODULE_MSG);
 
-	msg->extension = SOF_IPC4_MOD_EXT_MSG_PARAM_ID(SOF_IPC4_GAIN_PARAM_ID);
+	/* volume controls with range 0-1 (on/off) are switch controls */
```

Comment:
can we add a comment on what 0 and 1 mean? it's a classic recipe for confusions with 1 means muted or on...

Reply:
It is the value directly from ALSA layer, let me try to find the definition from there and copy/refer to it.

Reply:
that's precisely the point, is the convention ALSA-centric and in this case 0 means mute, or possibly another convention defined by IPC4 legacy.

Reply:
it'd be fine to add just a comment to make sure this is captured for future generations.

Reply:
For switch, 0 is off and 1 is on.